docs/plugins/woodpecker-plugins/src/index.ts

```diff
@@ -29,6 +29,23 @@ async function loadContent(): Promise<Content> {
           return undefined;
         }
 
+        let pluginIcon: string | undefined;
+        let pluginIconType: string | undefined;
+        if (docsHeader.icon) {
+          try {
+            const response = await axios(docsHeader.icon, {
+              responseType: 'arraybuffer',
+            });
```

Comment:
What do you think about storing the image as data-url directly?

```suggestion
            const response = await axios(docsHeader.icon, {
              responseType: 'blob',
            });
            
            pluginIconDataUrl = URL.createObjectURL(response.data);
```